### PR TITLE
Add direct connect link to server booking message

### PR DESCRIPTION
**Pull request by AndKenneth**

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -295,6 +296,14 @@ function DemosURL(name) {
     return `${settings.ssc.demo_root_path}/${settings.ssc.client}/${encodedName}`;
 }
 
+function DirectLink(string) {
+    let [ip, password, rcon] = string.split(';');
+    ip = ip.trim().split(' ')[1]
```
Comment (BenjaminSchaaf) [inline src/app.js]: Add `;` at end of line

File: src/app.js [hunk 2]
```diff
@@ -295,6 +296,14 @@ function DemosURL(name) {
...
+    password = password.trim().split(' ')[1].replace(/"/g, '')
```
Comment (BenjaminSchaaf) [inline src/app.js]: use `substring` instead of `replace`

**Review by BenjaminSchaaf — CHANGES_REQUESTED** [review]
After changes lgtm

File: src/app.js (hunk 1 above)
Comment (AndKenneth) [reply src/app.js]: rip my python experience.

File: src/app.js [hunk 3]
```diff
@@ -298,10 +298,10 @@ function DemosURL(name) {
 
 function DirectLink(string) {
     let [ip, password, rcon] = string.split(';');
-    ip = ip.trim().split(' ')[1]
-    password = password.trim().split(' ')[1].replace(/"/g, '')
+    ip = ip.trim().split(' ')[1];
+    password = password.trim().split(' ')[1].slice(1,-1);
```
Comment (BenjaminSchaaf) [inline src/app.js]: Put a space in here: `slice(1, -1)`

**Comment by BenjaminSchaaf** [comment]
Local test succeeded, good work.

**Review by BenjaminSchaaf — APPROVED** [review]

**Pull request merged**